### ISSUE TITLE
Will not work on Windows

### ISSUE THREAD

**Issue by LyubomirT** [Kokonico/cMusic#3]
As far as I know, `termios` is a package only available at UNIX, an import of which will result in an error on any Python distribution built for Windows. Perhaps this can be replaced with something cross-platform instead?

**Comment by Kokonico** [comment]
yes, I've been trying to work on windows support. I mostly develop on a MacBook and don't know the windows terminal and overall dev experience. so it'll take some time for me to do it.
